lib/polo/adapters/postgres.rb

```diff
@@ -0,0 +1,19 @@
+module Polo
+  module Adapters
+    module Postgres
+      def on_duplicate_key_update(inserts, records)
+        raise 'Not implemented for Postgres'
```

Comment:
Maybe we should be a little more specific here.
Let's change it to: `on_duplicate: :override is not currently supported in the PostgreSQL adapter`